smd/main.go

```diff
@@ -77,26 +102,140 @@ func (o *options) resolve() (operation, error) {
 
 	switch {
 	case o.validatePath != "":
+		if o.merge || o.compare {
```

Comment:
The fact that you're using the case statement in a specific order is sort of confusing (certainly is very counter-intuitive to me since I don't see a switch statement as a succession of if conditions). I don't see how this is different from a `if else if else if`?

Reply:
improved slightly